ci-operator/config/opendatahub-io/ai-edge/opendatahub-io-ai-edge-main.yaml

```diff
@@ -46,6 +46,36 @@ tests:
         requests:
           cpu: 500m
     workflow: hypershift-hostedcluster-workflow
+- as: test-ai-edge-periodic
+  cron: 0 * * * *
```

Comment:
This is just for testing. We have to first merge it to check that the Slack notifications work.